packages/@okta/vuepress-site/docs/guides/submit-oin-app/main/saml2/backward-compatible-eg.md

```diff
@@ -3,7 +3,7 @@
    To handle empty `companyId` values, you can define the ACS URL as:
 
    ```bash
-   ${empty org.companyId ? 'https://login.myapp.io' : 'https://login.myapp.io?connection=' + org.companyId}
+   https://${empty org.companyId ? 'login.myapp.io' : 'login.myapp.io?connection=' += org.companyId}
    ```
 
-   This expression handles both scenarios where `companyId` is populated or empty.
\ No newline at end of file
+   This expression handles both scenarios where `companyId` is populated or empty.
```

Comment:
Huh, I wonder why there's a diff here. End-of-file newline, maybe?

Reply:
Maybe? Let me review the file again.

Reply:
Looks like that removed the highlighted change. Thanks for the tip @jackward-okta !